src/k8s/pkg/k8sd/types/cluster_config_test.go

```diff
@@ -43,21 +45,39 @@ func TestValidateCIDR(t *testing.T) {
 	// Create a new BootstrapConfig with default values
 	validConfig := types.ClusterConfig{
 		Network: types.Network{
-			PodCIDR: "10.1.0.0/16,2001:0db8::/32",
+			PodCIDR:     "10.1.0.0/16,2001:0db8::/32",
+			ServiceCIDR: "10.152.183.0/16",
 		},
 	}
-
 	err := validConfig.Validate()
 	g.Expect(err).To(BeNil())
 
-	// Create a new BootstrapConfig with invalid CIDR
-	invalidConfig := types.ClusterConfig{
-		Network: types.Network{
-			PodCIDR: "bananas",
-		},
+	// Test invalid cidrs
+	for _, tc := range []struct {
+		cidr string
+	}{
+		{cidr: "bananas"},
+		{cidr: "fd01::/64,fd02::/64,fd03::/64"},
+	} {
+		t.Run(tc.cidr, func(t *testing.T) {
+			g := NewWithT(t)
+			badPodCIDR := types.ClusterConfig{
+				Network: types.Network{
+					PodCIDR: tc.cidr,
+				},
+			}
+			err := badPodCIDR.Validate()
+			g.Expect(err).ToNot(BeNil())
+
+			badSvcCIDR := types.ClusterConfig{
+				Network: types.Network{
+					ServiceCIDR: tc.cidr,
+				},
+			}
+			err = badSvcCIDR.Validate()
+			g.Expect(err).ToNot(BeNil())
+		})
```

Comment:
Note: instead of comments, the test names can tell the "story" instead

```suggestion
	t.Run("InvalidCIDR", func(t *testing.T) {
		for _, tc := range []struct {
			cidr string
		}{
			{cidr: "bananas"},
			{cidr: "fd01::/64,fd02::/64,fd03::/64"},
		} {
			t.Run(tc.cidr, func(t *testing.T) {
				t.Run("Pod", func(t *testing.T) {
					g := NewWithT(t)
					config := types.ClusterConfig{
						Network: types.Network{
							PodCIDR: tc.cidr,
						},
					}
					err := config.Validate()
					g.Expect(err).ToNot(BeNil())
				})
				t.Run("Service", func(t *testing.T) {
					g := NewWithT(t)
					config := types.ClusterConfig{
						Network: types.Network{
							ServiceCIDR: tc.cidr,
						},
					}
					err := config.Validate()
					g.Expect(err).ToNot(BeNil())
				})
			})
		}
	})
```

Reply:
NICE